cartridges/int_algolia_sfra/cartridge/static/default/js/algolia/instantsearch-config.js

```diff
@@ -410,7 +420,6 @@ function enableInstantSearch(config) {
                                     return variant.variantID === item.defaultVariantID;
                                 }) || item.variants[0];
                             }
-
```

Comment:
I prefer to keep those newlines, the function is quite long so it helps visually.